specs/core/0_beacon-chain.md

```diff
@@ -92,6 +92,15 @@ The primary source of load on the beacon chain are "attestations". Attestations
 | `ENTRY` | `0` |
 | `EXIT` | `1` |
 
+**Domains for BLS signatures**
+
+| Name | Value | 
+| - | :-: |
+| `DOMAIN_DEPOSIT` | `0` |
+| `DOMAIN_ATTESTATION` | `1` |
+| `DOMAIN_PROPOSAL` | `2` |
```

Comment:
@vbuterin I just merged `master` branch (includes `PROPOSER_SLASHING` stuff) into this PR, so you probably want to update using `DOMAIN_PROPOSAL` stuff in this PR?